src/common.h

```diff
@@ -90,6 +90,10 @@
 #define PRIaddress "u"
 #define PRIoffset PRIzx
 
+typedef struct v128_data {
```

Comment:
This is C++, so you can just do `struct v128 { ... };`

Reply:
Agree.